Tools/CISupport/ews-build/events.py

```diff
@@ -117,11 +139,32 @@ def buildStarted(self, key, build):
         builder = yield self.master.db.builders.getBuilder(build.get('builderid'))
         builder_display_name = builder.get('description')
 
+        sha = self.getProperty(build, 'revision')
```

Comment:
For all these events, it might be better/cleaner to create two methods one for ews and one for github. Maybe something like: sendBuildStartedDataToEWS, sendBuildStartedDataToGitHub(), and have buildStarted() call them appropriately.

Reply:
Does that apply only to `BuildStarted` or to all of them? I was trying to avoid a ton of function bloat in this class, if we did this everywhere, we'd be changing 4 functions to into 12.

Reply:
Actually, I just removed the code in `BuildStarted` completely. The reason I did this is that we really don't want to report anything until we validate the patch is actually relevant, I haven't made that change in it's entirety yet, but given we're reporting on each intermediate step, it shouldn't be terribly difficult to do.